scripts/insert_into_appointments.sql

```diff
@@ -1 +1,8 @@
--- insert appointments creating connections between doctors and patients
+INSERT INTO appointments(date, time, comments)
+SELECT 'August 12, 2016', '10AM', p.id, d.id
```

Comment:
FYI, sql is very specific about the string format it accepts for `date` and `time`. I would encourage you to go look at the documentation for that.